src/applications/prometheus/controller/PhabricatorPrometheusMetricsController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Prometheus\CollectorRegistry;
+use Prometheus\RenderTextFormat;
+use Prometheus\Storage\InMemory as InMemoryStorage;
```

Comment:
Not quite sure how this is supposed to be imported into phabricator. I suspect I need to do something additional.